<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>output_edit_dialog</class>
 <widget class="QDialog" name="output_edit_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>493</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>tuna.gui.output.edit.dialog.title</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tuna.qrc">
    <normaloff>:/images/icons/obs.png</normaloff>:/images/icons/obs.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>tuna.gui.tab.basics.song.info</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="txt_path"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>tuna.gui.tab.basics.song.format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>tuna.gui.tab.basics.format.info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>%t</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>tuna.gui.tab.basics.format.title</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>%e</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>tuna.gui.tab.basics.format.linebreak</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>%m</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>tuna.gui.tab.basics.format.artist</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>%d</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>tuna.gui.tab.basics.format.disc.number</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>%n</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>tuna.gui.tab.basics.format.number</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>%a</string>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>tuna.gui.tab.basics.format.album</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>%r</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>tuna.gui.tab.basics.format.date.full</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>%p</string>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>tuna.gui.tab.basics.format.time.progress</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>%y</string>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string>tuna.gui.tab.basics.format.date.year</string>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string>%l</string>
        </property>
       </item>
       <item row="4" column="3">
        <property name="text">
         <string>tuna.gui.tab.basics.format.time.length</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>%b</string>
        </property>
       </item>
       <item row="5" column="1">
        <property name="text">
         <string>tuna.gui.tab.basics.format.label</string>
        </property>
       </item>
       <item row="5" column="2">
        <property name="text">
         <string>%s</string>
        </property>
       </item>
       <item row="5" column="3">
        <property name="text">
         <string>tuna.gui.tab.basics.format.space</string>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>%o</string>
        </property>
       </item>
       <item row="6" column="1">
        <property name="text">
         <string>tuna.gui.tab.basics.format.time.left</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_format"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_logmode">
       <property name="text">
        <string>tuna.gui.output.edit.dialog.logmode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tuna.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>output_edit_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>output_edit_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
